libs/client/features/src/onboarding/steps/setup/AddFirstAccount.tsx

```diff
@@ -59,6 +59,8 @@ export function AddFirstAccount({ title, onNext }: StepProps) {
                                             loader={BrowserUtil.enhancerizerLoader}
                                             src={`financial-institutions/white/${src}.svg`}
```

Comment:
These assets are missing so they won't load during onboarding. @Shpigford do you have these or should we plan to replace them?